Responsive Design Improvements with HTML Adjustments

jeelan-jd opened the pull request:
This PR makes the design more responsive across different screen sizes. It includes changes to the HTML structure to ensure that the layout looks good and functions well on mobile, tablet, and desktop devices. The updates focus on better alignment, readability, and user interaction, making the site easier to use on any device.

jeelan-jd committed:
responsive design changes

iamharis010 commented:
Thanks @jeelan-jd for looking into it. I noticed that, on small screen thickness circle get's higher hight. It should be in circle shape.

jeelan-jd committed:
circle small device design changes

jeelan-jd commented:
Please check again for small device circle design in responsive.

iamharis010 committed:
- Code formatting Added.
- Open/Close Issue of download button resolved.
- Keep disable the download button if there is no signature on pad

iamharis010 reviewed (approved):
Seems okay to me now.
